samples/bot-conversation/csharp/Bots/TeamsConversationBot.cs

```diff
@@ -50,6 +52,8 @@ protected override async Task OnMessageActivityAsync(ITurnContext<IMessageActivi
                 await CardActivityAsync(turnContext, true, cancellationToken);
             else if(text.Contains("message"))
                 await MessageAllMembersAsync(turnContext, cancellationToken);
+            else if(text.Contains("immersivereader"))
```

Comment:
We need entry point for to be added from welcome message. 

See this: 
https://github.com/OfficeDev/Microsoft-Teams-Samples/pull/642#discussion_r1058286867

Reply:
Yes added it